relayer/crates/starknet-test-components/src/impls/bootstrap/bootstrap_chain.rs

```diff
@@ -58,6 +59,10 @@ where
             .await
             .map_err(Bootstrap::raise_error)?;
 
+        // FIXME: RPC address is set to localhost and port is set to a random free port
+        // The values should be configurable to connect to a specific node
```

Comment:
For bootstrapping, the host should always be localhost since we are running the chain process. If we are moving to testnet, we will just use a dummy bootstrap context that returns a chain with preconfigured address.